drivers/hmis/lib/form_data/default/fragments/signature.json

```diff
@@ -0,0 +1,100 @@
+{
+  "type": "GROUP",
+  "variant": "SIGNATURE",
+  "link_id": "signature_group",
+  "text": "eSignature",
+  "subtitle": "The following signatures are required on this document.",
+  "item": [
+    {
+      "type": "GROUP",
+      "text": "Head of Household",
+      "link_id": "head_of_household_signature_group",
+      "item": [
+        {
+          "type": "STRING",
```

Comment:
One result of having these be separate inputs is that I don't think there's any existing example in `DynamicField` of having 2 inputs next to each other on a line when they are separated like this in the json. The comparable examples I could find, such as the Name input, use `"type": "OBJECT"`. Maybe we should take that approach here, as well? (this approach could also help us out with the question of how to display the checkbox if it doesn't correspond to any field on the backend.)